server/routes/bookedtoday/arrivals/noRecordFoundController.ts

```diff
@@ -8,6 +10,16 @@ export default class NoRecordFoundController {
     return async (req, res) => {
       const { id } = req.params
       const data = await this.expectedArrivalsService.getArrival(id)
+
+      State.newArrival.set(res, {
+        firstName: convertToTitleCase(data.firstName),
+        lastName: convertToTitleCase(data.lastName),
+        dateOfBirth: data.dateOfBirth,
+        sex: data.gender,
+        prisonNumber: data.prisonNumber,
+        pncNumber: data.pncNumber,
+      })
```

Comment:
this should be done in a post handler - same as the single match one should be moved to

Reply:
though I guess this will move to the next page so NBD